### PR TITLE
integrity: load module

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/integritysetup/integritysetup-generator.c [hunk 1]
```diff
@@ -85,6 +85,7 @@ static int create_disk(
                 "DefaultDependencies=no\n"
                 "IgnoreOnIsolate=true\n"
                 "After=integritysetup-pre.target systemd-udevd-kernel.socket\n"
+                "After=modprobe@dm_integrity.service\n"
```
Comment (poettering) [inline src/integritysetup/integritysetup-generator.c:88]: hmm, why do we have to load the manually? normally module auto-loading by the DM layer should suffice? why not here?

Comment (yuwata) [reply src/integritysetup/integritysetup-generator.c:88]: Hmm. Not sure. But the reporter said that the original issue can be fixed by adding the module to modules-load.d.

**Comment by keszybz** [comment]
The patch looks OK, but it'd be nice to figure out why we need this at all. Let's wait for the logs in the bugzilla.

**Comment by poettering** [comment]
I cherry-picked the first commit directly into git main.

Judging by the latest rhbz comment this is just an selinux issue, it blocks the automatic loading, and really shouldn't.

Hence let's close this, as we should not tape over that.

**Comment by yuwata** [comment]
Ok. Sure. Thank you for talking a look.

**Pull request closed without merging**